### DISCUSSION TITLE
What are clnx files? Using clna or clns are giving error in postanalysis

### DISCUSSION THREAD

**Discussion by meg-gits-it** [milaboratory/mixcr#1186] [General]
Documentation for running postanalysis for CDR3 metrics, diversity, V/J/VJ-usage, CDR3/V-Spectratype metrics.
      (cloneset.(clns|clna)|directory)...
                           Paths to input clnx files or to directories with clnx files. Files in directories will not be filtered by extension.

Error: Require clnx file type, got /Output1/Sample1_L001.clna...

**Comment by mizraelson** [comment]
Hi,
Can you please share the the set of mixcr commands you used for data analysis and the software version.

**Reply by meg-gits-it** [reply]
The documentation for Individual Postanalysis says-
mixcr postanalysis individual 
  --default-downsampling (<type>|none) 
  --default-weight-function (<read>|<Tag>|none) 
  [--only-productive] 
  [--drop-outliers] 
  [--chains <chain>[,<chain>...]]... 
  [--metadata <path>] 
  [--group <group>]... 
  [--tables <path.(tsv|csv)>] 
  [--preproc-tables <path.(tsv|csv)>] 
  [-O <key=value>]... 
  [--force-overwrite] 
  [--no-warnings] 
  [--verbose] 
  [--help] 
  (cloneset.(clns|clna)|directory)... result.json[.gz]

where Command line options:

(cloneset.(clns|clna)|directory)...
Paths to input clnx files or to directories with clnx files. Files in directories will not be filtered by extension.

result.json[.gz]
Path where to write postanalysis result.

--default-downsampling (<type>|none)
Default [downsampling](https://mixcr.com/mixcr/reference/mixcr-postanalysis/#downsampling) applied to normalize the clonesets. Possible values: count-[read|TAG]-[auto|min|fixed][-<number>], top-[read|TAG]-[<number>], cumtop-[read|TAG]-[percent], none

--default-weight-function (<read>|<Tag>|none)
Default clonotype [weight function](https://mixcr.com/mixcr/reference/mixcr-postanalysis/#weight-functions)

--drop-outliers
Drop samples which are below downsampling value as computed according to specified default downsampling option.

I used the following for my BCR analysis providing the path to my Output files from mixcr analyze command. 

mixcr postanalysis individual  --default-downsampling count-read-auto-400 --default-weight-function none --drop-outliers --verbose /Output1/Sample1_L001.clna ... /Postanalysis

Error:
Require clnx file type, got /Output1/Sample1_L001.clna...

**Reply by meg-gits-it** [reply]
MiXCR v4.3.2

**Comment by meg-gits-it** [comment]
The problem got solved by removing the three dots ' ... '
